Tools/WinMLRunner/BindingUtilities.h

```diff
@@ -65,10 +71,29 @@ namespace BindingUtilities
             auto stream = file.OpenAsync(FileAccessMode::Read).get();
             // Create the decoder from the stream
             BitmapDecoder decoder = BitmapDecoder::CreateAsync(stream).get();
-            // get the bitmap
-            SoftwareBitmap softwareBitmap = decoder.GetSoftwareBitmapAsync(TypeHelper::GetBitmapPixelFormat(inputDataType), BitmapAlphaMode::Ignore).get();
-            
-            return softwareBitmap;
+
+            // If input dimensions are different from tensor input, then scale / crop while reading
+            if (decoder.PixelHeight() != height ||
+                decoder.PixelWidth() != width)
+            {
+                // Create a transform object with default parameters (no transform)
+                auto transform = BitmapTransform();
+                transform.ScaledHeight(height);
+                transform.ScaledWidth(width);
+                transform.InterpolationMode(BitmapInterpolationMode::Cubic);
```

Comment:
Is it a good idea to hardcode this interpolation mode in the transform? Would it be scenario dependent?

Reply:
Possibly, my best attempt to get the model the best possible input.  Could make it part of the flag?

Reply:
Yeah let's make it an argument that's required when passing in the flag. Sounds good?